### PR TITLE
Added numero2/contao-proper-filenames

### PR DESCRIPTION
**Pull request by bennyborn**

### CODE REVIEW

**Comment by richardhj** [comment]
Hey :)
1) You don't need the multiline syntax with `>`. Just type the description after the colon (`:`)
2) Can you please add to the description that this extension is for file uploads in the _back end_ via the file manager?
3) Do you want to add a title like "Proper upload filenames"?

Feel free to follow any advice if you want to.

**Comment by bennyborn** [comment]
Hi Richard,

1. OK, will keep it simple
2. Will be done in the upcoming commit :)
3. I felt like leaving this out intentionally since the package name itself is already pretty descriptive

Any advice on my little travis problem? Should I just add the term "filenames" to the whitelist?

**Comment by richardhj** [comment]
>     3. I felt like leaving this out intentionally since the package name itself is already pretty descriptive

The title is more "natural" language.

> Any advice on my little travis problem? Should I just add the term "filenames" to the whitelist?

You can use "file name" or add "filename" (singular) to the whitelist if necessary.

File: meta/numero2/contao-proper-filenames/de.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+de:
+    title: Ordentliche Dateinamen
+    description: Ersetzt Sonderzeichen in Dateinamen bei Dateien welche über die Dateiverwaltung im Backend hochgeladen wurden.
```
Comment (xchs) [inline meta/numero2/contao-proper-filenames/de.yml]:
```suggestion
    description: Ersetzt Sonderzeichen in Dateinamen, welche über die Dateiverwaltung im Contao Backend hochgeladen wurden.
```

File: meta/numero2/contao-proper-filenames/en.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en:
+    title: Proper file names
+    description: Replaces special characters in the name of a file right after uploading it in the backend using the file manager.
```
Comment (xchs) [inline meta/numero2/contao-proper-filenames/en.yml]:
```suggestion
    description: Replaces special characters in the name of a file right after uploading it in the Contao back end using the file manager.
```

**Comment by richardhj** [comment]
I wanted to revert the changes you've made in the dictionary. Most importantly, "backend" is not a correct entry in the English dictionary. However, I do not have push permissions to your branch, so please do a
```
git remote add upstream git@github.com:contao/package-metadata.git
git fetch --all
git checkout upstream/master -- linter/whitelists/en.txt
git add .
git commit -m "Revert dictionary changes"
git push
```

**Pull request merged**